doc/source/reference/arrays.datetime.rst

```diff
@@ -100,7 +100,9 @@ Datetime and Timedelta Arithmetic
 NumPy allows the subtraction of two Datetime values, an operation which
 produces a number with a time unit. Because NumPy doesn't have a physical
 quantities system in its core, the timedelta64 data type was created
-to complement datetime64.
+to complement datetime64. The arguments for timedelta64 are a number, 
+to represent the number of units, and a date/time unit, such as
+(D)ay, (M)onth, (Y)ear, (h)ours, (m)inutes, or (s)econds.
```

Comment:
Both datetime64 and timedelta64 also accept "NAT" (in any combination of upper and lower case) in place of the number for a "Not A Time" value. An example with that would be good and few words about it would be a nice addition. Also this page has a section "Differences Between 1.6 and 1.7 Datetimes". Please remove it.

Reply:
If you feel unsure about what I asked for let me know, it can be challenging to format some of the documentation.

Reply:
@mattip I am having trouble generating the documentation. After installing all the dependencies, I successfully built numpy from scratch with `python setup.py install`. 

However, when running `make html`, I run into
![image](https://user-images.githubusercontent.com/24601452/64743069-557cfd80-d4b3-11e9-8f6d-78d6dc513666.png)

Is there a step I am missing?

Reply:
You need to downgrade sphinx to 1.8.5, we don’t yet support sphinx 2.

Reply:
@mattip sorry about the trouble, I am trying to learn as I go, but I ran into another problem when generating the docs with `make html`.

Can you please help me out? Here's the log file:
```
# Sphinx version: 1.8.5
# Python version: 3.7.1 (CPython)
# Docutils version: 0.15.2 release
# Jinja2 version: 2.10.1
# Last messages:
#   writing output... [  0%] dev/index
#   
#   writing output... [  0%] dev/releasing
#   
#   writing output... [  0%] dev/style_guide
#   
#   writing output... [  0%] docs/howto_build_docs
#   
#   writing output... [  0%] docs/howto_document
#   
# Loaded extensions:
#   sphinx.ext.mathjax (1.8.5) from /anaconda3/lib/python3.7/site-packages/sphinx/ext/mathjax.py
#   alabaster (0.7.12) from /anaconda3/lib/python3.7/site-packages/alabaster/__init__.py
#   sphinx.ext.autodoc (1.8.5) from /anaconda3/lib/python3.7/site-packages/sphinx/ext/autodoc/__init__.py
#   sphinx.ext.autosummary (1.8.5) from /anaconda3/lib/python3.7/site-packages/sphinx/ext/autosummary/__init__.py
#   numpydoc (0.9.0.dev0) from /Users/andrewliu/Projects/numpy/doc/sphinxext/numpydoc/__init__.py
#   sphinx.ext.intersphinx (1.8.5) from /anaconda3/lib/python3.7/site-packages/sphinx/ext/intersphinx.py
#   sphinx.ext.coverage (1.8.5) from /anaconda3/lib/python3.7/site-packages/sphinx/ext/coverage.py
#   sphinx.ext.doctest (1.8.5) from /anaconda3/lib/python3.7/site-packages/sphinx/ext/doctest.py
#   sphinx.ext.graphviz (1.8.5) from /anaconda3/lib/python3.7/site-packages/sphinx/ext/graphviz.py
#   sphinx.ext.ifconfig (1.8.5) from /anaconda3/lib/python3.7/site-packages/sphinx/ext/ifconfig.py
#   matplotlib.sphinxext.plot_directive (3.1.1) from /anaconda3/lib/python3.7/site-packages/matplotlib/sphinxext/plot_directive.py
#   IPython.sphinxext.ipython_console_highlighting (unknown version) from /anaconda3/lib/python3.7/site-packages/IPython/sphinxext/ipython_console_highlighting.py
#   IPython.sphinxext.ipython_directive (unknown version) from /anaconda3/lib/python3.7/site-packages/IPython/sphinxext/ipython_directive.py
#   sphinx.ext.imgmath (1.8.5) from /anaconda3/lib/python3.7/site-packages/sphinx/ext/imgmath.py
#   sphinx.ext.linkcode (1.8.5) from /anaconda3/lib/python3.7/site-packages/sphinx/ext/linkcode.py
Traceback (most recent call last):
  File "/anaconda3/lib/python3.7/site-packages/sphinx/cmd/build.py", line 304, in build_main
    app.build(args.force_all, filenames)
  File "/anaconda3/lib/python3.7/site-packages/sphinx/application.py", line 341, in build
    self.builder.build_update()
  File "/anaconda3/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 347, in build_update
    len(to_build))
  File "/anaconda3/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 412, in build
    self.write(docnames, list(updated_docnames), method)
  File "/anaconda3/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 593, in write
    self._write_serial(sorted(docnames))
  File "/anaconda3/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 604, in _write_serial
    self.write_doc(docname, doctree)
  File "/anaconda3/lib/python3.7/site-packages/sphinx/builders/htmlhelp.py", line 246, in write_doc
    StandaloneHTMLBuilder.write_doc(self, docname, doctree)
  File "/anaconda3/lib/python3.7/site-packages/sphinx/builders/html.py", line 731, in write_doc
    self.docwriter.write(doctree, destination)
  File "/anaconda3/lib/python3.7/site-packages/docutils/writers/__init__.py", line 78, in write
    self.translate()
  File "/anaconda3/lib/python3.7/site-packages/sphinx/writers/html.py", line 58, in translate
    self.document.walkabout(visitor)
  File "/anaconda3/lib/python3.7/site-packages/docutils/nodes.py", line 178, in walkabout
    if child.walkabout(visitor):
  File "/anaconda3/lib/python3.7/site-packages/docutils/nodes.py", line 178, in walkabout
    if child.walkabout(visitor):
  File "/anaconda3/lib/python3.7/site-packages/docutils/nodes.py", line 178, in walkabout
    if child.walkabout(visitor):
  [Previous line repeated 1 more time]
  File "/anaconda3/lib/python3.7/site-packages/docutils/nodes.py", line 170, in walkabout
    visitor.dispatch_visit(self)
  File "/anaconda3/lib/python3.7/site-packages/docutils/nodes.py", line 1912, in dispatch_visit
    return method(node)
  File "/anaconda3/lib/python3.7/site-packages/sphinx/writers/html.py", line 886, in visit_math_block
    visit(self, node)
  File "/anaconda3/lib/python3.7/site-packages/sphinx/ext/imgmath.py", line 323, in html_visit_displaymath
    fname, depth = render_math(self, latex)
  File "/anaconda3/lib/python3.7/site-packages/sphinx/ext/imgmath.py", line 252, in render_math
    dvipath = compile_math(latex, self.builder)
  File "/anaconda3/lib/python3.7/site-packages/sphinx/ext/imgmath.py", line 139, in compile_math
    p = Popen(command, stdout=PIPE, stderr=PIPE)
  File "/anaconda3/lib/python3.7/subprocess.py", line 769, in __init__
    restore_signals, start_new_session)
  File "/anaconda3/lib/python3.7/subprocess.py", line 1516, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
NotADirectoryError: [Errno 20] Not a directory: 'latex'
```

Also, the documentation https://numpy.org/devdocs/docs/howto_build_docs.html doesn't give information to download Sphinx version <2, so would it be a good idea to add it?